privacy_base/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    'name': 'Data Privacy and Protection',
+    'version': '9.0.1.0.0',
+    'category': 'Data Protection',
+    'summary': 'Provides data privacy and protection features '
+               'to comply to regulations, such as GDPR.',
```

Comment:
This addon provides none of those features, just a base for them.

Better something like: *Base for privacy-related addons*.

Reply:
I'm thinking on how it will show in the Apps list.